__tests__/app.test.js

```diff
@@ -423,6 +423,30 @@ describe("article queries", () => {
   });
 });
 
+describe("get article by id returns comment count", () => {
+  it("200: resolves with article", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        const article = body.article[0];
```

Comment:
You're responding with an array of data - when you're only sending a single object it's best practice to just send the object on a key - to avoid users having to use `[0]`